src/core/lib/experiments/experiments.yaml

```diff
@@ -72,14 +72,14 @@
   test_tags: ["core_end2end_test", "event_engine_client_test"]
 - name: monitoring_experiment
```

Comment:
could we do something about this experiment so we don't have to keep pushing it out?

Reply:
yes we can, but i think a different pr (or set of them tbh)

Reply:
ok, sg.